earth_enterprise/src/fusion/autoingest/tools/geaddtoimageryproject.cpp

```diff
@@ -105,6 +109,24 @@ main(int argc, char *argv[]) {
     if (help) {
       usage(progname);
     }
+    
+    if (overridemax && overridemax_deprecated) {
+      throw khException("Cannot use maxlevel and maxleveloverride together, " 
+                       "maxlevel is deprecated.\n");
+    }
+
+    if (overridemax) {
+      if (AssetType == AssetDefs::Imagery) {
+        overridemax = ImageryToProductLevel(overridemax);
+      }
+      else {
+        overridemax = TmeshToProductLevel(overridemax);
+      }
+    }
+    
+    if (overridemax_deprecated) {
+      overridemax = overridemax_deprecated;
```

Comment:
I suggest that we output a message here letting the user know that the command is deprecated and suggest the other one instead.

Reply:
Added a deprecated message.